main.js

```diff
@@ -0,0 +1,247 @@
+const display = document.getElementById("display");
+const start = document.querySelector(".start");
+const input = document.querySelector("#input");
+document.querySelector("button").onclick = myClick;
+
+display.innerHTML = "00:00";
+
+let timeOut;
+let time = myClick;
+function myClick() {
+  time = document.querySelector("#input").value;
+  console.log(time);
+}
+
+start.addEventListener("click", function () {
+  timeOut = setInterval(timers, 1000);
+});
+
+function timers() {
+  if (time <= 0) {
+    window.location.reload();
+  } else {
+    let minutes = Math.floor(time / 60);
+    minutes = minutes < 10 ? "0" + minutes : minutes;
+    let seconds = time % 60;
+    seconds = seconds < 10 ? "0" + seconds : seconds;
+    display.innerHTML = `${minutes}:${seconds}`;
+    time--;
+  }
+}
+
+// Task.2
+
+const confirm = document.querySelector("#confirm");
+const days = document.querySelector("#days");
+const hours = document.querySelector("#hours");
+const minutes = document.querySelector("#minutes");
+const seconds = document.querySelector("#seconds");
+document.querySelector(".confirm").onclick = inputeDateTransform;
+
+const currentDate = new Date();
+
+function inputeDateTransform() {
+  let dateEnd = document.querySelector("#input-data").value;
+  dateEnd = Date.parse(dateEnd);
+}
+
+function myCalendarTimer() {

```

Comment:
There is also no validation, so the user able to see NaN as a timer 


https://user-images.githubusercontent.com/84010501/200114631-9b53d849-c2dc-4864-b3e4-9cf62ca1e472.mov